src/serac/numerics/functional/tests/functional_basic_h1_vector.cpp

```diff
@@ -150,20 +160,26 @@ void test_suite(std::string meshfile)
   }
 }
 
+TEST(VectorValuedH1, test_suite_hexes) { test_suite("/data/meshes/patch3D_hexes.mesh"); }
+
+#ifndef SERAC_USE_CUDA_KERNEL_EVALUATION
 TEST(VectorValuedH1, test_suite_tris) { test_suite("/data/meshes/patch2D_tris.mesh"); }
 TEST(VectorValuedH1, test_suite_quads) { test_suite("/data/meshes/patch2D_quads.mesh"); }
 TEST(VectorValuedH1, test_suite_tris_and_quads) { test_suite("/data/meshes/patch2D_tris_and_quads.mesh"); }
-
 TEST(VectorValuedH1, test_suite_tets) { test_suite("/data/meshes/patch3D_tets.mesh"); }
-TEST(VectorValuedH1, test_suite_hexes) { test_suite("/data/meshes/patch3D_hexes.mesh"); }
 TEST(VectorValuedH1, test_suite_tets_and_hexes) { test_suite("/data/meshes/patch3D_tets_and_hexes.mesh"); }
+#endif
 
 int main(int argc, char* argv[])
 {
   int num_procs, myid;
 
   ::testing::InitGoogleTest(&argc, argv);
 
+#ifdef SERAC_USE_CUDA_KERNEL_EVALUATION
+  serac::accelerator::initializeDevice();
```

Comment:
is this missing a terminate?  Also, can we put the macro guards inside the function to avoid duplication?